pkg/controllers/updaterun/initialization_integration_test.go

```diff
@@ -753,7 +753,7 @@ func generateSucceededInitializationStatus(
 	updateStrategy *placementv1beta1.ClusterStagedUpdateStrategy,
 	clusterResourceOverride *placementv1alpha1.ClusterResourceOverrideSnapshot,
 ) *placementv1beta1.StagedUpdateRunStatus {
-	return &placementv1beta1.StagedUpdateRunStatus{
+	status := &placementv1beta1.StagedUpdateRunStatus{
```

Comment:
nit: we are reusing util functions across the different integration tests suites. Maybe we can move them all into a common test_util file just like the e2e does? Not blocking this PR.